{"uid":"5f00edf47c1156a9","name":"test_send_message_with_non_exist_user","fullName":"test_Front.TestNegative#test_send_message_with_non_exist_user","historyId":"dd476b1f3e4c4e6b621836f5c04db61b","time":{"start":1729954930119,"stop":1729954930119,"duration":0},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: Can't find element by locator ('xpath', '//button//div[text()=\"Вход\"]')\nStacktrace:\n#0 0x55d3b81600aa <unknown>\n#1 0x55d3b7c771a0 <unknown>\n#2 0x55d3b7cc3d06 <unknown>\n#3 0x55d3b7cc3fa1 <unknown>\n#4 0x55d3b7d09344 <unknown>\n#5 0x55d3b7ce7dad <unknown>\n#6 0x55d3b7d06866 <unknown>\n#7 0x55d3b7ce7b23 <unknown>\n#8 0x55d3b7cb6990 <unknown>\n#9 0x55d3b7cb796e <unknown>\n#10 0x55d3b812c16b <unknown>\n#11 0x55d3b812ff68 <unknown>\n#12 0x55d3b811964c <unknown>\n#13 0x55d3b8130ae7 <unknown>\n#14 0x55d3b80fe4af <unknown>\n#15 0x55d3b814e4f8 <unknown>\n#16 0x55d3b814e6c0 <unknown>\n#17 0x55d3b815ef26 <unknown>\n#18 0x7f7491a94ac3 <unknown>","statusTrace":"request = <SubRequest 'init_driver' for <Function test_send_message_with_non_exist_user>>\n\n    @pytest.fixture(scope=\"class\")\n    def init_driver(request):\n        # Инициализация драйвера\n        options = webdriver.ChromeOptions()\n        options.add_argument('--headless')\n        driver = webdriver.Chrome(options=options)\n    \n        driver.implicitly_wait(10)\n        driver.maximize_window()\n        request.cls.driver = driver\n    \n        # Инициализация страницы\n        login_page = LoginPage(driver)\n        home_page = HomePage(driver)\n    \n        # Открытие страницы\n        login_page.open_login_page()\n    \n        # Логинимся\n>       login_page.login_user()\n\nconftest.py:29: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \nlogin_page.py:27: in login_user\n    self.find_element(self.login_user_button_locator())\nbase_page.py:13: in find_element\n    return WebDriverWait(self.driver, time).until(EC.presence_of_element_located(locator),\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"d3d0e4bfc098c8863096cf45b45b483a\")>\nmethod = <function presence_of_element_located.<locals>._predicate at 0x7f991c4dc4c0>\nmessage = 'Can\\'t find element by locator (\\'xpath\\', \\'//button//div[text()=\"Вход\"]\\')'\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: Can't find element by locator ('xpath', '//button//div[text()=\"Вход\"]')\nE       Stacktrace:\nE       #0 0x55d3b81600aa <unknown>\nE       #1 0x55d3b7c771a0 <unknown>\nE       #2 0x55d3b7cc3d06 <unknown>\nE       #3 0x55d3b7cc3fa1 <unknown>\nE       #4 0x55d3b7d09344 <unknown>\nE       #5 0x55d3b7ce7dad <unknown>\nE       #6 0x55d3b7d06866 <unknown>\nE       #7 0x55d3b7ce7b23 <unknown>\nE       #8 0x55d3b7cb6990 <unknown>\nE       #9 0x55d3b7cb796e <unknown>\nE       #10 0x55d3b812c16b <unknown>\nE       #11 0x55d3b812ff68 <unknown>\nE       #12 0x55d3b811964c <unknown>\nE       #13 0x55d3b8130ae7 <unknown>\nE       #14 0x55d3b80fe4af <unknown>\nE       #15 0x55d3b814e4f8 <unknown>\nE       #16 0x55d3b814e6c0 <unknown>\nE       #17 0x55d3b815ef26 <unknown>\nE       #18 0x7f7491a94ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/selenium/webdriver/support/wait.py:105: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"init_driver","time":{"start":1729954930119,"stop":1729954943405,"duration":13286},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: Can't find element by locator ('xpath', '//button//div[text()=\"Вход\"]')\nStacktrace:\n#0 0x55d3b81600aa <unknown>\n#1 0x55d3b7c771a0 <unknown>\n#2 0x55d3b7cc3d06 <unknown>\n#3 0x55d3b7cc3fa1 <unknown>\n#4 0x55d3b7d09344 <unknown>\n#5 0x55d3b7ce7dad <unknown>\n#6 0x55d3b7d06866 <unknown>\n#7 0x55d3b7ce7b23 <unknown>\n#8 0x55d3b7cb6990 <unknown>\n#9 0x55d3b7cb796e <unknown>\n#10 0x55d3b812c16b <unknown>\n#11 0x55d3b812ff68 <unknown>\n#12 0x55d3b811964c <unknown>\n#13 0x55d3b8130ae7 <unknown>\n#14 0x55d3b80fe4af <unknown>\n#15 0x55d3b814e4f8 <unknown>\n#16 0x55d3b814e6c0 <unknown>\n#17 0x55d3b815ef26 <unknown>\n#18 0x7f7491a94ac3 <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/_pytest/setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 1140, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/_pytest/fixtures.py\", line 891, in call_fixture_func\n    fixture_result = next(generator)\n  File \"/home/runner/work/Dc/Dc/conftest.py\", line 29, in init_driver\n    login_page.login_user()\n  File \"/home/runner/work/Dc/Dc/login_page.py\", line 27, in login_user\n    self.find_element(self.login_user_button_locator())\n  File \"/home/runner/work/Dc/Dc/base_page.py\", line 13, in find_element\n    return WebDriverWait(self.driver, time).until(EC.presence_of_element_located(locator),\n  File \"/opt/hostedtoolcache/Python/3.10.15/x64/lib/python3.10/site-packages/selenium/webdriver/support/wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true}],"afterStages":[],"labels":[{"name":"tag","value":"@pytest.mark.usefixtures('init_driver', 'base_url')"},{"name":"suite","value":"test_Front"},{"name":"subSuite","value":"TestNegative"},{"name":"host","value":"fv-az847-117"},{"name":"thread","value":"2284-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test_Front"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"tags":["@pytest.mark.usefixtures('init_driver', 'base_url')"]},"source":"5f00edf47c1156a9.json","parameterValues":[]}